app/assets/stylesheets/ama_layout/layout_components/accordions.scss

```diff
@@ -76,6 +76,105 @@
   }
 }
 
+.accordion-multi-state{
+  @extend .accordion;
+  margin-bottom: $base-margin;
+  background: transparent;
+
+  .accordion-item{
+    margin-bottom: $base-margin;
+  }
+
+  &__header{
+    @extend .accordion__header;
+    background: transparent;
+    position: relative;
+  }
+
+  &__heading-bg{
+    @extend .accordion__heading;
+    padding: 0.63rem 1rem !important;
```

Comment:
Sure, the only reason I didnt do that is if we want to change the background colour of all our accordions we'd have to change every class.